### PR TITLE
Couple of fixes for xcode12 support

### PR DESCRIPTION
**Pull request by ollitapa**
Hi, sorry if I'm a bit too eager to get SPM-support for this project, but seemed like this branch missed some finishing touches. I had to fix the Bundle search function to match what the SPM generates to `Bundle.module`, because it was not finding the correct bundle on my app. Hopefully this supports CocoaPods/Carthage too, but I did not test it.

### CODE REVIEW

File: SwiftMessages/NSBundle+Utils.swift [hunk 1]
```diff
@@ -8,20 +8,31 @@
...
-        }
-        // Check for CocoaPods or Carthage bundle name
-        else if let path = bundle.path(forResource: "SwiftMessages", ofType: "bundle") {
-            return Bundle(path: path)!
-        }
-        // Just return the app bundle
-        else {
-            return bundle
+
+        let bundleName = "SwiftMessages_SwiftMessages"
```
Comment (wtmoose) [inline SwiftMessages/NSBundle+Utils.swift:17]: This isn't the right bundle name for CocoaPods, so it won't work as-is

File: Package.swift [hunk 2]
```diff
@@ -15,6 +15,10 @@ let package = Package(
             path: "SwiftMessages",
             exclude: [
                 "Info.plist",
+            ],
+            resources: [
```
Comment (wtmoose) [inline Package.swift:19]: Is it necessary to add this item? In my test scenarios it was not.

Comment (ollitapa) [reply Package.swift:19]: Ah, maybe not, tried this before I found out that the bundle was wrong.

File: SwiftMessages/NSBundle+Utils.swift (hunk 1 above)
Comment (ollitapa) [reply SwiftMessages/NSBundle+Utils.swift:17]: Ok, I was skeptical of that too 😅 I'll try if I can modify the original a little less to get the correct bundle in SPM case.

File: Package.swift (hunk 2 above)
Comment (ollitapa) [reply Package.swift]: Removed and works!

**Comment by wtmoose** [comment]
How about the adjustment I pushed? I tried to streamline the implementation a bit with a nested loop. And I returned a fallback value that matches the original implementation.

There's a separate issue with CocoaPods, but I can handle that after merging this into the feature branch.

**Comment by ollitapa** [comment]
Looks good! I can confirm it still works on my project. 👍

**Comment by wtmoose** [comment]
Great! Gonna merge this in and try to get a release out. Thanks for helping out

**Pull request merged**